proposals/lip-0045.md

```diff
@@ -340,28 +391,20 @@ channelSchema = {
                 },
                 "root": {
                     "dataType": "bytes",
+                    "length": HASH_LENGTH,
                     "fieldNumber": 3
                 }
             }
         },
         "partnerChainOutboxRoot": {
             "dataType": "bytes",
+            "length": HASH_LENGTH,
```

Comment:
You write below:

>The value of this property is set to the outbox root computed from the last CCU from the partner chain. 

Isn't it rather "obtained" from the last CCU then "computed" from the last CCU?

Reply:
you compute it if there are ccms attached in the ccu

Reply:
I though you only ever copy the `outboxRoot` from the CCU to the `partnerChainOutboxRoot` and only the inbox root is something updated via computation.

Reply:
Okay, can be resolved.